### PR TITLE
[debian] Update changelog

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit updates changelog for release.

Related: #10540 
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: infra/debian/compiler/changelog [hunk 1]
```diff
@@ -1,3 +1,16 @@
+one (1.21.0) bionic; urgency=medium
```
Comment (seanshpark) [inline infra/debian/compiler/changelog:1]: 1.21.0 exist below.

Comment (mhs4670go) [reply infra/debian/compiler/changelog:1]: OH,, Thank you!

**Review by jinevening — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**